.changeset/angry-penguins-flow.md

```diff
@@ -0,0 +1,7 @@
+---
+'@sap-cloud-sdk/connectivity': minor
+'@sap-cloud-sdk/test-util': minor
```

Comment:
I feel this is not touched?

Reply:
I confused the `test-util` from the test-resources with the test-util we ship. Removed